### ISSUE TITLE
Update ConfigurationOfSeaside3

### ISSUE THREAD

**Issue by IwanVosloo** [SeasideSt/Seaside#981]
There was a bugfix release 3.2.4, but the release3.2 symbolic version in [ConfigurationOfSeaside3>>release32..st](https://github.com/SeasideSt/Seaside/blob/v3.2.4/repository/ConfigurationOfSeaside3.package/ConfigurationOfSeaside3.class/instance/release32..st)  is pointing to 3.2.2 still.

**Comment by theseion** [comment]
Thank you. We'll take care of it.

**Comment by jbrichau** [comment]
We should discontinue updating that repo and redirect everybody to github.

**Comment by jbrichau** [comment]
@marschall I propose we just discontinue updating the old ConfigurationOf on Smalltalkhub. I did this during a brief period so people can load using the 'old' style.

In the pharo7 branch, I already updated the readme a bit to better reflect this. I could already merge that one into master. https://github.com/SeasideSt/Seaside/commit/13744868b79e1a9ec6043bdeed2d441c8b634d26#diff-04c6e90faac2675aa89e2176d2eec7d8

Any thoughts?
